buf/registry/module/v1beta1/upload_service.proto

```diff
@@ -42,7 +42,7 @@ message UploadRequest {
     //
     // If the ModuleRef refers to a Module that has associated Content, this field should *not*
     // be set, and setting it is an error.
-    string commit_id = 2 [(buf.validate.field).string.uuid = true];
+    optional string commit_id = 2 [(buf.validate.field).string.uuid = true];
```

Comment:
`optional` means something very specific, namely that this field will have field presence. I don't think we need presence here, the default value (empty string) should suffice, and it does in various other places in the API: https://github.com/bufbuild/registry-proto/blob/8054b81b8e70e42eeed9ffd50fd1ad29af78ab1a/buf/registry/legacy/federation/v1beta1/resource.proto#L77-L78

We could update the documentation to clarify this.

Reply:
Sure. @emcfarlane agrees with you. I don't have a preference tbh, I thought it doesn't make sense to send and empty value here and instead the requester shouldn't set it, but alright I'll do the change and update docs. 👍🏼